apps/docs/stories/components/SezzlePaymentMethod/Docs.mdx

```diff
@@ -0,0 +1,44 @@
+import { Meta, Title, ArgTypes, Source } from '@storybook/blocks';
+import { filterDocsByTag, ExportedParts } from '../../utils';
+import { codeExampleFull } from './example';
+import { ComponentBox } from '../../utils/documentation-components';
+import * as JustifiSezzlePaymentMethod from './index.stories';
+import { setUpMocks } from '../../utils/mockAllServices';
+
+{setUpMocks()}
+
+<Meta
+  title="Modular Checkout/Sezzle Payment Method"
+  Components="justifi-sezzle-payment-method"
+  of={JustifiSezzlePaymentMethod}
+/>
+
+<Title />
+
+---
+
+Renders a radio input for selecting the Sezzle payment method, if BNPL is enabled for the account.
+
+This component is **designed to be used within** the `justifi-modular-checkout` and **does not accept props directly**. Instead, it relies on shared state passed through the Stencil Store, managed by the wrapper component.
```

Comment:
Edit to ' ... on the shared state...' and '... managed by the `justifi-modular-checkout` component.'